app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/ytdliability/taxable_gain_or_loss.scala.html

```diff
@@ -122,17 +123,24 @@
     @caption(messages("ytdLiability.caption"))
 
     @formWrapper(routes.YearToDateLiabilityController.taxableGainOrLossSubmit()) {
+      <h1 class="govuk-heading-xl">@title</h1>
```

Comment:
you might need a div with an id here so that the unit tests can find the content